src/Lights/Shadows/babylon.shadowGenerator.ts

```diff
@@ -605,21 +605,26 @@
         public forceCompilation(onCompiled: (generator: ShadowGenerator) => void, options?: { useInstances: boolean }): void {
             let shadowMap = this.getShadowMap();
             if (!shadowMap) {
+                onCompiled(this);

```

Comment:
On compiled could be null (even if I forgot to set it to Nullable)

Reply:
Okay, I'll make both Nullable then.